src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php

```diff
@@ -384,14 +386,20 @@ private function isUrlValid(string $url, string $needle): bool
     /**
      * Compare strings case insensitive.
      *
-     * @param array  $metadataElement   Doi metadata elements.
-     * @param string $comparisonElement String that needs to be compared.
+     * @param array        $metadataElement   Doi metadata elements.
+     * @param string       $comparisonElement String that needs to be compared.
+     * @param Dataset|null $dataset           A dataset instance.
      *
      * @return void
      */
-    private function doesStringExist(array $metadataElement, string $comparisonElement): void
+    private function doesStringExist(array $metadataElement, string $comparisonElement, Dataset $dataset = null): void
     {
         if (empty($metadataElement[$metadataElement['field']])) {
+            if (!empty($dataset)
+                and $dataset->getAvailabilityStatus() === DatasetSubmission::AVAILABILITY_STATUS_NOT_AVAILABLE
+            and $metadataElement['field'] === 'author') {
```

Comment:
Does this need to be tabbed over?

Reply:
Yes, I don't know why GCS didn't catch it